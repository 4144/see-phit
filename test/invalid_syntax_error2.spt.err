if(eos) ParseError("Unexpected end of stream"); if(empty_tag) ParseError("Expecting an identifier"); ParseError("Unexpected character inside tag content"); ParseError("Missing <"); ParseError("Missing </"); ParseError("Expecting a tag name after <"); ParseError("Expecting an open tag at top level");
