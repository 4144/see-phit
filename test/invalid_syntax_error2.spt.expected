../seephit.h:697:7:   in constexpr expansion of 'ParseError(((const char*)"Expecting a close tag")).ErrLine::operator[](((size_t)iLine))'
invalid_syntax_error2.spt:1:1: error: array subscript value '4' is outside the bounds of array type 'int [0]'
